### PR TITLE
Math magicians: fetch data from API

### PR DESCRIPTION
**Pull request by jojo987N**
###  Math magicians: fetch data from API
For this milestone :

- Create a new component for displaying a quote.
- Use the following external API Ninjas
- Fetch data inside of a new component
- Show a loading state
- Import the new component into main component
- Check that there are no linter errors
- Use correct Gitflow

### CODE REVIEW

File: src/components/calculator/calculator.css [hunk 1]
```diff
@@ -3,7 +3,7 @@
   grid-template-columns: 1fr 1fr 1fr 1fr;
 
   /* border: 1px solid green; */
```
Comment (rustam-tolipov) [inline src/components/calculator/calculator.css]: **You did great job! The app is working as expected :)**

- [ ] But in your project, I can see there are some cleaning jobs. Kindly delete all unused commented codes to keep your app professional

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
 // import logo from './logo.svg';
```
Comment (rustam-tolipov) [inline src/App.js]: - [ ] Kindly remove this code as well :)

File: src/components/quote/Quote.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  useEffect(() => {
+    (async () => {
+      const res = await fetch(`${URL_BASE}=happiness`, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          'X-Api-Key': API_KEY,
+        },
+      });
+      const json = await res.json().catch(() => setHasError(true));
```
Comment (rustam-tolipov) [inline src/components/quote/Quote.js]: [OPTIONAL] It would be good idea if you used `try..catch` method to catch errors :)

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @jojo987N,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Well-written readme_
_Good design_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @jojo987N, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] The `useEffect` hook has been used correctly ✔️ 
- [ ] There is a `loading` state ✔️ 
- [ ] There is an `error` state ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**